azure-functions-maven-plugin/src/test/java/com/microsoft/azure/maven/function/invoker/CommonUtils.java

```diff
@@ -35,10 +37,21 @@ public static void deleteAzureResourceGroup(String resourceGroupName, boolean wa
                         waitForOperationFinish ? "" : " --no-wait"));
     }
 
-    private static void executeCommand(final String command) throws IOException, InterruptedException {
+    public static String executeCommand(final String command) throws IOException, InterruptedException {
```

Comment:
better to add javadoc for this public function, because from the name `executeCommand` I cannot easily know what is the returned string. (from the code I suppose it returns the output of the command though)